src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
-		<TargetFrameworks>net7.0;net7.0-windows10.0.18362;net7.0-android;net7.0-ios;net7.0-maccatalyst;net7.0-macos</TargetFrameworks>
```

Comment:
This change is another breaking change that should be combined with the previous one (change to .NET 7). It should be listed in the BREAKING_CHANGES.md file.

Reply:
This is already list in BREAKING_CHANGES. This file wasn't been update on last PR.